packages/cron/README.md

```diff
@@ -0,0 +1,105 @@
+# Create a Cron Job with Functionless
+
+In this example, we'll create a Cron job.
+
+The Cron Job will use [AWS EventBridge's](https://aws.amazon.com/eventbridge/) scheduled rules to trigger a job in a lambda [Lambda Functions](https://docs.aws.amazon.com/lambda/latest/dg/welcome.html).
+
+Then we'll deploy and test the application.
+
+To see the completely application, go [here](https://github.com/functionless/functionless-samples/packages/cron).
+
+## Create an new app
+
+> yarn create functionless
+
+For a more detailed guide, see: https://functionless.org/docs/getting-started/setup
+
+## Create the cron job
+
+Replace your `app.ts` with (the imports will be useful later):
+
+```ts
+import { App, Duration, Stack } from "aws-cdk-lib";
+import { Schedule } from "aws-cdk-lib/aws-events";
+import { EventBus, Function } from "functionless";
+
+const app = new App();
+const stack = new Stack(app, "functionlessCron");
+
+EventBus.schedule(stack, "cron", Schedule.rate(Duration.minutes(1)));
+```
+
+This create a rule which executes any targets every minutes. Lets give it something to do.
+
+## Metaphors on the Minute
```

Comment:
I don't understand this heading?

Reply:
Stupid joke. The API I'm using returns metaphors.